### ISSUE TITLE
^B macro doesn't work after new release

### ISSUE THREAD

**Issue by Farzat07** [neomutt/neomutt#3356]
Previously I had this line in my configuration file:
```
macro pager \cb <pipe-entry>'urlscan'<enter> 'Follow links with urlscan'
```

However, after updating to the new version `NeoMutt 20220408`, that no longer works. When I press ctrl-b, nothing happens, and as the next button is pressed it outputs the following message: `Key is not bound.  Press '?' for help.`.

It seems like this version comes with its own default implementation; when I comment out my own macro, it tries to run urlview, which is not installed on my system. Interestingly, urlview is called with no issues, but once I add my own macro the binding stops working, and neither urlscan nor urlview are called.

Basically I have two questions:

- Is there a reason urlview is used? urlscan is the one I know of, but I am interested to know if there better ones.
- Regardless of the above question, how do I fix the above issue? Or alternatively, how can I make ctrl-b open urlscan or another tool instead of urlview?

**Comment by scottkosty** [comment]
I like to use urlscan also since it is actively maintained.

On current master, the following macro works for me:

```
macro index,pager \cb "<pipe-message> urlscan --compact<Enter>" "call urlscan to extract URLs out of a message"
```

**Comment by flatcap** [comment]
Sorry about that.
As Scott notes, it's already fixed.
There will be a new release later this week.

If you're not easily scared, you can read [what went wrong](https://github.com/neomutt/neomutt/issues/3343#issuecomment-1094271206).

**Comment by Farzat07** [comment]
Amazing. Works on both the release and master branch. Thanks a lot!
